---
layout: page
title: "News"
subheadline: "About"
permalink: "/about_news/"
header:
    image_fullwidth: "header_mps.png"
---

### 6/1/20: Christina's review on interpretable machine learning is published

<img src="/images/news/200601_chrisitna-tig-cover.jpg" width=100 align=right>In collaboration with Jiliang Tang from Comp. Sci. & Engr, MSU, Christina's review on [__Opening the Black Box: Interpretable Machine Learning for Geneticists__](https://www.ncbi.nlm.nih.gov/pubmed/32396837) is now published in Trends in Genetics. This work highlights current progress in making machine learning model interpretable and explanable using applications in genetics as examples. And her graphics was picked as the cover as shown on the right!

### 3/31/20: Collaborative work with Dr. Yan Bao is published

Dr. Bao's work on [__COST1 regulates autophagy to control plant drought tolerance__](https://www.ncbi.nlm.nih.gov/pubmed/32170020) is published where Peipei has contributed significantly using comparative genomics approaches.



### 3/19/20: Siobhan defended today

<img src="/images/news/200319_siobahn-before-denfense.png" width=100 align=right>Siobahn defended her thesis titled _Modeling and prediction of genetic redundancy in Arabidopsis thaliana and Saccharomyces cerevisiae_ today __virtually__ via Zoom with 89 people in the audience! Both us showed up in the lab (while keeping >6 ft distance) to get the virtual defense setup. Originally due to move to England next week, Siobhan will stay in East Lansing to finish her work. Good job Siobhan! The picture on the right shows Siobhan right after the defense.

### 3/11/20: Lab COVID-19 contigency plan issued

University is deciding the timing to suspend face-to-face instruction. Even if the suspension is in place, instruction will continue online. To prepare for this, When the University suspend face-to-face instruction, all computational research individuals will work remotely (well, this was announced 2 hours after the lab plan). 

1. For lab personnel, only essential operations (e.g. keep plants watered, as defined by Melissa) will remain.
2. Prior to 1), should you feel the need to work remotely, please inform Shinhan so he is aware.
3. All scheduled lab functions (lab meeting and KATSUP) will be maintained, except a remote option will be offered with details following.
4. For any lab personnel engaged in independent research projects (specifically Peipei, Serena, Thilanka) and working remotely should meet with Shinhan daily for 15-30 min to discuss goals for the day.  Such plan should be documented in Trello and shared with Shinhan.
5. For any lab personnel mentoring others or being mentored (e.g. Peipei and Thilanka, Thilanka and Patricia, Melissa and Abigail), make sure you come up with means and frequency of communication to ensure projects are moving forward.

For online communication, including lab meeting, KATSUP, and project discussion, Zoom will be used.

### 3/5/20: Welcome to the lab Ally

<img src="/images/people/Ally.png" width=100 align=right>Ally has decided to join the Plant Biology Program and, instead of doing any rotation, is coming to our lab! She is currently finishing up her Bachelor's degree in Manchester University, a liberal art institution. She is awarded an MSU College of Natural Science Fellowship as well as an Early-Start Fellowship to begin her doctoral training on July 1st! Welcome to our lab Ally!

### 2/21/20: Welcome to the lab Serena

<img src="/images/people/Serena.png" width=100 align=right>Serena is a new graduate student in the, hold your breath, Plant Biology Program, the Computational Math., Sci., Math Program, AND the Moleculr Plant Science Program. She is a MSU Plant Science Fellow and an NSF Fellow of the IMPACTS training program. She graduated from the Cornell University in 2019 and has been rotating in two other labs. Below is the surprising Slack exchange when I found out she is joining us:
<img src="/images/news/200221_serena-slack.png" width=100 align=right>

### 10/25/19: Beth defended her PhD

<img src="/images/news/191025_beth-before-defense.png" width=100 align=right>Beth defended her PhD thesis today titled _Specilized metabolism and stress response: studies in predicting gene function and regulation_. The seminar was held in Plant Biology Lab 247 and was attended by both her colleagues, friends, and families. Beth is moving to University of Wisconsin-Madison to take a postdoctoral position applying molecular evolutionary and computaional approaches to plant biochemistry. We wish her all the best! The picture on the right shows Beth right before her defense.

### 10/20/19: Ronan is here!

Siobhan gave birth today to her baby boy Ronan! Congrats Siobhan and Geoff!

### 9/27/19: Christina defeneded her PhD

<img src="/images/news/191003_christina-last-day.png" width=100 align=right>Christina today defended her PhD thesis titled _Interpretable machine laerning in plant genomes: studies in the complex relationship between genotype and phenotype_. The seminar was held in Plant Biology Lab 247 and was full! Christina's parents also attended her seminar. In the talk, Christina laid out the overarching Christina is now moving to Australia working in algorithm development in single cell genomics studies. All the best Christina! The picture on the right showing Chrisitna leaving the lab for good on 10/3/2020...

### 9/24/19: Sahra and Christina's work on the regulatory mechanisms of cell-type stress response is published

Our story titled [__Cis-regulatory code for predicting plant cell-type transcriptional response to high salinity__](https://www.ncbi.nlm.nih.gov/pubmed/31551359) is now published in Plant Physiology with Sahra and Christina as joint first authors. Using the root cell-type transcriptome data, Sahra and Christina identified cis-regulatory sequences likely specify cell-type response to high salinity stress. More importantly, machine learning models were built to ask how well the identified regulatory sequences can predict cell-type response. The findings not only advance our understanding of the regulatory mechanisms of the plant spatial transcriptional response, but also suggest broad applicability of the approach to any species, particularly those with little or no trans regulatory data.

### 9/18/19: Christina's work on benchmarking genomic prediction algorithms is published

Chrisitina's paper on [__Benchmarking Parametric and Machine Learning Models for Genomic Prediction of Complex Traits__](https://www.ncbi.nlm.nih.gov/pubmed/31533955) is published today in G3. This is a collaborative between our colleague Gustavo de los Compos, Chrisitna's internship mentors Andrew McCarren and Mark Roantree, and Emily Bolger a highly productive summer research student. Using data of 18 traits across six plant species with different marker densities and training population sizes, Christina spearheaded a comparison of the performance of six linear and six non-linear algorithms.  The findings highlight that simpler, linear algorithm can beat out more sophiscated machine learning approaches, and the importance of algorithm selection for the prediction of trait values.

### 8/20/19: John's work on poaceae intergenic transcription is published

John's paper on [__Evolutionary characteristics of intergenic transcribed regions indicate rare novel genes and widespread noisy transcription in the Poaceae__](https://www.ncbi.nlm.nih.gov/pubmed/31431676) is published in Scientific Report. Christina as well as Rosalie Sowers, a highly talented undergrad from Penn State also contribute to this work. Extensive transcriptional activity occurring in intergenic regions of genomes has raised the question whether intergenic transcription represents the activity of novel genes or noisy expression. To assess this, John et al. evaluated cross-species and post-duplication sequence and expression conservation of intergenic transcribed regions. This study provides a framework to identify novel genes using comparative transcriptomic data to improve genome annotation that is fundamental for connecting genotype to phenotype in crop and model systems.

### 8/19/19: Our newest addition to the lab, Thilanka

<img src="/images/people/thilanka.jpg" width=100 align=right>Thilanka is a new graduate student in the Plant Biology Graduate Program. He graduated from the University of Peradeniya in Sri Lanka in 2016 and worked as a teaching assistant in the past two years in Peradeniya. He is energetic, communicative, and excited about learning new topics. He is also easy to get alone with, and has a black belt in Taekwondo!! Welcome to the lab!

### 7/24/19: Siobhan got the Dissertation Completion Fellowship

Siobhan has been selected to receive a $7,500 College of Natural Science Dissertation Completion Fellowship during fall semester 2019. Congrats Siobhan!

### 7/13/19: Nick's work on regulatory assymmetry of plant duplicate genes is published

Nick's paper on [__Expression and regulatory asymmetry of retained Arabidopsis thaliana transcription factor genes derived from whole genome duplication__](https://www.ncbi.nlm.nih.gov/pubmed/30866803) is published in BMC Evoltionary Biology. Christina and Eamon Winship, an undergraduate reseracher at the time also contributed. Transcription factors (TFs) play a key role in gene regulation and tend to be retained after duplication. Nick et al.'s work provide answers about about how TF duplicates have diverged in their expression and regulation that contribute to a better understanding of the elevated retention rate among TFs.

### Lots happened before this point.. But we transition to Github after July 2019, so...
