---
layout: page
subheadline:  "Publications"
title:  ">= 5 years old"
teaser: "<sup>e<sub>: joint first/corresponding. <sup>u<sub>:undergrad. <sup>h<sub>:high school student. <sup>v<sub>:visiting scholar. Bold: people from the Shiu lab."
header:
   image_fullwidth: "header_homepage_13.jpg"
permalink: "/publications_more_than_5_years/"
---

[Click here for publications in the last 5 years](https://shiulab.github.io/publications_in_last_5_years/)

## 2014

**Panchy N**, **Wu G**, Newton L, Tsai CH, Chen J, Benning C, Farre EM<sup>e<sub>, **Shiu SH**<sup>e<sub> (2014) Prevalence, Evolution, and *cis*-Regulation of Diel Transcription in *Chlamydomonas reinhardtii*. *G3* 4(12):2461-2471 [Find Here](http://www.ncbi.nlm.nih.gov/pubmed/25354782)




**Azodi CB**, McCarren A, Roantree M, de los Campos G, **Shiu SH**. Benchmarking algorithms for genomic prediction of complex traits. *G3* DOI: 10.1534/g3.119.400498 [Find Here](https://www.g3journal.org/content/early/2019/09/18/g3.119.400498)

**Azodi CB**, Pardo J, VanBuren R, de los Campos G, **Shiu SH**. Transcriptome-based prediction of complex traits in maize. *The Plant Cell* DOI: 10.1101/587121 [In press](https://www.biorxiv.org/content/early/2019/03/27/587121.full.pdf)

**Uygun S[e]**, **Azodi, CB[e]**, **Shiu, SH**. Cis-regulatory code for predicting plant cell-type specific high salinity response. *Plant Physiol.* DOI: 10.1104/pp.19.00653 [Find Here](http://www.plantphysiol.org/content/early/2019/09/24/pp.19.00653.long)

**Lloyd JP**, Bowman MJ, **Azodi CB**, Moghe GD, Childs KL, **Shiu SH**. Evolutionary characteristics of intergenic transcribed regions indicate rare novel genes and widespread noisy transcription in the Poaceae. *Scientific Reports* 9:12122 [Find Here](https://www.nature.com/articles/s41598-019-47797-y)

**Panchy NL**, **Azodi CB**, Winship EF, **Shiu SH** (2019) Expression and regulatory asymmetry of retained *Arabidopsis thaliana* transcription factor genes derived from whole genome duplication. *BMC Genomics* 19:77 [Find Here](https://bmcevolbiol.biomedcentral.com/articles/10.1186/s12862-019-1398-z)

Chang YM, Lin HH, Liu WY, Yu CP, Chen HJ, Kao YY, Wu YH, Lin JJ, Li MYJ, Tu SL, Wu SH, **Shiu SH**, Ku MSB, Li WH (2019) A comparative transcriptomics method to infer gene coexpression networks and its applications to maize and rice leaf transcriptomes. *Proc. Natl. Acad. Sci., USA* 116(8):3091-3099 [Find Here](https://www.ncbi.nlm.nih.gov/pubmed/30718437)

**Moore BM**, **Wang P**, Fan P, Leong B, Schenck C, **Lloyd J**, **Lehti-Shiu MD**, Last R, Pichersky E, **Shiu SH**(2019) Robust predictions of specialized metabolism genes through machine learning. *Proc. Natl. Acad. Sci., USA* 116(6):2344-2353 [Find Here](https://www.ncbi.nlm.nih.gov/pubmed/30674669)

## 2018

**Wang P**, **Moore BM**, **Panchy NL**, **Meng F**, **Lehti-Shiu MD**, **Shiu SH** (2018) Factors influencing gene family size variation among related species in a plant family, Solanaceae. *Genome Biol. Evol.* 10(10):2596–2613 [Find Here](https://www.ncbi.nlm.nih.gov/pubmed/29743197)

**Liu MJ**, Sugimoto K, **Uygun, S**, **Panchy NL**, Campbell M, Yandell M, Howe GA, **Shiu SH** (2018) Regulatory Divergence In Wound-Responsive Gene Expression In Domesticated And Wild Tomato. *Plant Cell* 30(7):1445-1460 [Find Here](https://www.ncbi.nlm.nih.gov/pubmed/29743197)

**Lloyd JP**, **Tsai ZTY**, Sowers RP[u], **Panchy NL**, **Shiu SH** (2018) A model-based approach for identifying functional intergenic transcribed regions and non-coding RNAs. *Mol. Biol. Evol.* 35(6):1422-1436 [Find Here](https://www.ncbi.nlm.nih.gov/pubmed/29554332)

Tsai CH, **Uygun S**, Roston R, **Shiu SH**, Benning C (2018) Recovery from N Deprivation Is a Transcriptionally and Functionally Distinct State in Chlamydomonas. *Plant Physiol.* 176(3):2007-2023 [Find Here](https://www.ncbi.nlm.nih.gov/pubmed/29288234) 

## 2017

Huang CF, Yu CP, Wu YH, Lu MJ, Tu SL, Wu SH, **Shiu SH**, Ku MSB, Li WH (2017) Elevated auxin biosynthesis and transport underlie high vein density in C4 leaves. *Proc. Natl. Acad. Sci., USA* 114(33):E6884 [Find Here](https://www.ncbi.nlm.nih.gov/pubmed/28761000)

**Uygun S[e]**, **Seddon AE[e]**, **Azodi C**, **Shiu SH** (2017) Predictive models of spatial transcriptional response to high salinity. *Plant Physiol.* 174:450-464 [Find Here](https://www.ncbi.nlm.nih.gov/pubmed/28373393) 

**Tsai ZTY**, **Lloyd J**, **Shiu SH** (2017) Defining functional, genic regions in the human genome through integration of biochemical, evolutionary, and genetic evidence. *Mol. Biol. Evol.* 34(7):1788-1798 [Find Here](https://www.ncbi.nlm.nih.gov/pubmed/28398576)

Bennett MS, **Shiu SH**, Triemer RE. (2017) A rare case of plastid protein-coding gene duplication in the chloroplast genome of Euglena archaeoplastidiata (Euglenophyta). *J Phycol.* 53(3):493-502 [Find Here](https://www.ncbi.nlm.nih.gov/pubmed/28295310)

**Lehti-Shiu MD[e]**, **Panchy N**, **Wang P**, **Uygun S**, **Shiu SH[e]**. (2017) Diversity, expansion, and evolutionary novelty of plant DNA-binding transcription factor families. *Biochim. Biophys. Acta Gene Regul. Mech.* 1860(1):3-20 [Review] [Find Here](http://www.ncbi.nlm.nih.gov/pubmed/27522016)

## 2016

**Uygun S**, Peng C, **Lehti-Shiu MD**, Last R, **Shiu SH**. (2016) Utility and limitations of using gene expression data to identify functional associations. *PLoS Comp. Biol.* 12(12):e1005244 [Find Here](https://www.ncbi.nlm.nih.gov/pubmed/27935950)

**Panchy N**, **Lehti-Shiu MD**, **Shiu SH**. (2016) Evolution of Gene Duplication in Plants. *Plant Physiol.* 171(4):2294-316. [Review] [Find Here](http://www.ncbi.nlm.nih.gov/pubmed/27288366)

Kang H, Zhu D, Lin R, Opiyo SO, Jiang N, **Shiu SH**, Wang GL. (2016) A novel method for identifying polymorphic transposable elements via scanning of high-throughput short reads. *DNA Research* 23(3):241-51. [Find Here](http://www.ncbi.nlm.nih.gov/pubmed/27098848)

Liu T, Newton L, **Liu MJ**, **Shiu SH**, Farre EM. (2016) A G-box-like motif is necessary for transcriptional regulation by circadian pseudo-response regulators in Arabidopsis. *Plant Physiol.* 170(1):528-39. [Find Here](http://www.ncbi.nlm.nih.gov/pubmed/26586835)

## 2015

**Lloyd JP**, **Seddon AE**, **Moghe GD**, **Simenc MC[u]**, **Shiu SH** (2015) Characteristics of plant essential genes allow for within- and between-species prediction of lethal mutant phenotypes. *Plant Cell* 27:2133 [Find Here](http://www.ncbi.nlm.nih.gov/pubmed/26286535)

**Tsai ZTY**, **Shiu SH[e]** and Tsai HK[e] (2015) Contribution of sequence motif, chromatin state, and DNA structure features to predictive models of transcription factor binding in yeast. *PLoS Comp. Biol.* 11(8):e1004418 [Find Here](http://www.ncbi.nlm.nih.gov/pubmed/26291518)

**Liu MJ**, **Seddon AE**, **Tsai ZTY**, Major IT, Floer M, Howe GA, **Shiu SH** (2015) Determinants of nucleosome positioning and their influence on plant gene expression. *Genome Res.* 25(8):1182-95. [Find Here](http://shiulab.plantbiology.msu.edu/images/8/80/2015_GenomeRes_Liu_nucleosome.pdf)

Poliner E, **Panchy N**, Newton L, **Wu G**, Lapinsky A, Bullard B, Zienkiewicz A, Benning C, **Shiu SH**, Farré EM. (2015) Transcriptional coordination of physiological responses in Nannochloropsis oceanica CCMP1779 under light/dark cycles. *Plant J* 83:1097-113. [Find Here](http://www.ncbi.nlm.nih.gov/pubmed/26216534)

Peng C, **Uygun S**, **Shiu SH**, Last R. (2015) The Impact of the Branched-Chain Ketoacid Dehydrogenase Complex on Amino Acid Homeostasis in Arabidopsis. *Plant Physiol*. 169(3):1807-20. [Find Here](http://www.ncbi.nlm.nih.gov/pubmed/25986129)

**Lehti-Shiu MD[e]**, **Uygun S**, **Moghe, GD**, **Panchy N**, **Fang L**, **Hufnagel D[u]**, **Jasicki HL[h]**, Feig M, **Shiu SH[e]**. (2015) Molecular evidence for functional divergence and decay of a transcription factor derived from whole genome duplication in Arabidopsis thaliana. *Plant Physiol.* 168(4):1717-34. [Find Here](http://www.ncbi.nlm.nih.gov/pubmed/26103993)

Yu CP, Chen SC, Chang YM, Liu WY, Lin HH, Lin JJ, Chen HJ, Lu YJ, Wu YH, Lu MY, Lu CH, Shih AC, Ku MS, **Shiu SH[e]**, Wu SH[e], Li WH[e]. (2015) Transcriptome dynamics of developing maize leaves and genome-wide prediction of cis elements and their cognate transcription factors. *Proc. Natl. Acad. Sci., USA* 112(19):E2477-86. [Find Here](http://www.ncbi.nlm.nih.gov/pubmed/25918418)

**Wu G**, **Hufnagel DE[u]**, **Denton AK[v]**, **Shiu SH** (2015) Retained duplicate genes in green alga Chlamydomonas reinhardtii tend to be stress response genes and experience frequent response gains. *BMC Genomics* 16:149. [Find Here](http://www.ncbi.nlm.nih.gov/pubmed/25880851)

Law M, Childs KL, Campbell MS, Stein D, Holt C, **Panchy N**, Lei J, Achawanantakun R, Jiao D, Andorf CM, Lawrence CJ, Ware D, **Shiu SH**, Sun Y, Jiang N, Yandell M (2015) Automated update, revision and quality control of the Zea mays genome annotations using MAKER-P improves the B73 RefGen_v3 gene models and identifies new genes. *Plant Physiol.* 167(1):25-39 [Find Here](http://www.ncbi.nlm.nih.gov/pubmed/25384563)

## Papers published >5 years ago

[Click here to see older publications.](http://shiulab.plantbiology.msu.edu/index.php?title=Publications_older)




