---
layout: page
subheadline:  "Research"
title:  "Projects"
teaser: "Check out what our lab is up to these days...</a>."
header:
   image_fullwidth: "header_homepage_13.jpg"
permalink: "/research_projects/"
---

# Evolution of genome contents
<img align="left" width="150" height="150" src="https://github.com/ShiuLab/ShiuLab.github.io/blob/master/images/research/research_cvalue.png"> 
Genome size variation in major plant taxonomic groups.There are 2,500 fold differences in plant genome size, ranging from ~60 million bases in Genlisea margaretae, a carnivorous plant, to 150 billion bases in Paris japonica, the canopy plant. Unlike other eukaryotes, plants also tend to have younger new genes in gene families than other species. With the increasing ease in sequencing and assessing expression of the entire genome of any species, many parts of the genomes that are between known genes are now found to be expressed. For more information, check out

- Our review paper discussing [evolution of duplicate genes in plants](https://www.ncbi.nlm.nih.gov/pubmed/27288366).
- Two Molecular Biology & Evolution papers integrating multi-omics datasets with machine learning methods to identify functional region [in Arabidopsis thaliana](https://www.ncbi.nlm.nih.gov/pubmed/29554332) and [in human genomes](https://www.ncbi.nlm.nih.gov/pubmed/28398576).


# Evolution of transcriptional response to stress
<img align="left" width="150" height="100" src="https://github.com/ShiuLab/ShiuLab.github.io/blob/master/images/research/research_cisreg.png"> 
Gain and loss of stress responses among duplicate genes.Under stressful environment, the expression of hundreds to thousands of genes changes at once. This response is essential for setting up proper physiological and developmental programs in plants so they can survive in adverse environment. Thus, proper response to stress is most likely under significant amount of selective pressure. 

Example publications:
- Our [Plant Physiology paper](https://www.ncbi.nlm.nih.gov/pubmed/27522016) detailing molecular evidence of functional decay in a transcription factor regulating stress response.
- Our [Plant Cell paper](https://www.ncbi.nlm.nih.gov/pubmed/29743197) on the evolution of stress response between domesticated and wild tomato species.


# Regulatory logic & systems biology
<img align="left" width="100" height="100" src="https://github.com/ShiuLab/ShiuLab.github.io/blob/master/images/research/research_cisreg.png"> 
How is gene expression regulated under stress? Substantial knowledge has accumulated on how short DNA motifs, cis-regulatory elements, in the promoter regions are involved in controlling gene expression. We are interested in deciphering the cis-regulatory code, i.e., how cis-elements work in concert to specify gene expression in response to stress. For more information on our research in this area, check out:

- Our [PNAS paper](https://www.ncbi.nlm.nih.gov/pubmed/21849619) analyzing stress expression data from Arabidopsis thaliana with systems biology approaches in the first comprehensive analysis of plant cis-regulatory code.
- Our PLoS Comp Biol paper discussing the utility and limitation in using expression data for identifying functionally related genes.
- Our Plant Physiology papers detailing predictive models of [plant organ](https://www.ncbi.nlm.nih.gov/pubmed/28373393) and [cell-type](https://www.ncbi.nlm.nih.gov/pubmed/31551359) transcriptional response to stress.


# Predictive biology with machine learning
Biology has become a data-rich discipline with a rapid influx of heterogeneous data in a rapid pace. Novel ways in looking at these data have the potential to answer new questions, make high quality predictions of many biological phenomenons through modeling, and, from predictive models, better understand the underlying mechanisms. Since our first foray into this area, our lab has applied machine learning methods in multiple contexts to predict molecular, physiological, and morphological phenotype. Below are some example publications in this area:

- Our study in [PLoS Computational Biology](https://www.ncbi.nlm.nih.gov/pubmed/26291518) on integrating sequence motif, chromatin state, and DNA structural information to predict transcription factor binding site in yeast.
- Our [PNAS paper](https://www.ncbi.nlm.nih.gov/pubmed/30674669) combining a large number of heterogeneous characteristics to predict whether an enzyme gene is involved in specialized metabolism or not.
- Our [Plant Cell paper](https://www.ncbi.nlm.nih.gov/pubmed/31641024) detailing a model for predicting maize complex traits using genome and transcriptome variation data
